doc/classes/TouchScreenButton.xml

```diff
@@ -5,6 +5,8 @@
 	</brief_description>
 	<description>
 		Button for touch screen devices. You can set it to be visible on all screens, or only on touch devices.
+		[TouchScreenButton] is different from a [Control] like [TextureButton] because it does not capture the input. It just activates when you slide a finger over it or press it.
```

Comment:
Could you clarify what "activates" means in this context. To me activating a button and capturing input seem like they could mean the same thing. Does activate mean active the button, but pass the input to other control nodes?

Reply:
That particular wording is from @reduz [here](https://github.com/godotengine/godot/issues/15039#issuecomment-353887848). But my understanding is that "activate the button, but pass the input to other control nodes" is the intended interpretation.

I think I found some other discussion that mentioned that the signal/event generated might also be different to `Button` but I don't have a link for that handy.

Reply:
I think we should clarify what activate means in the description. Because right now it risks confusing the user even more.